CNV#38773: Regional-DR with ODF for virt (#83970)
openshift/openshift-docs: modules/virt-vm-behavior-dr.adoc

Use relocate to move an application from the primary environment to the secondary environment when the primary environment is still accessible. During relocate, the VM is gracefully terminated, any unreplicated data is synchronized to the secondary environment, and the VM starts in the secondary environment.

> Because the VM terminates gracefully, there is no data loss in this scenario. Therefore, the VM operating system does not need to perform crash recovery.

Reviewer: The way it is written suggests that in other scenarios there might / will be data loss. Since that is a sensitive topic, we should be very clear about this.
To not make this paragraph too long I removed the "in this scenario" to avoid any comparisons between other scenarios.

```suggestion
Because the VM terminates gracefully, there is no data loss. Therefore, the VM operating system will not perform crash recovery.
```
